content/nginx-one/rbac/overview.md

```diff
@@ -0,0 +1,16 @@
+---
+description: 
+docs:
+doctypes:
+- reference
+tags:
+- docs
+title: "Overview: Role-based access control"
+toc: true
```

Comment:
None of the frontmatter is organised like newer documentation.

Reply:
I'm open to suggestions